### PR TITLE
Development

### PR DESCRIPTION
**Pull request by 1ba1**
Submitting bubble sort. Thanks in advance.

### CODE REVIEW

File: bubble_sort.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+def bubble_sort_by(a)
+  (a.length-1).times do
+    for i in 0...a.length - 1
+      a[i], a[i + 1] = a[i + 1], a[i] if yield(a[i], a[i + 1]) > 0
+    end
+
+  a
+end
+
```
Comment (heyapricot) [inline bubble_sort.rb:20]: The file is missing a couple of `end` statements

File: bubble_sort.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+def bubble_sort(a)
+  (a.length - 1).times do
+    for i in 0...a.length - 1
```
Comment (heyapricot) [inline bubble_sort.rb]: You can use the `..` operator instead (Two dots)

**Review by heyapricot — CHANGES_REQUESTED** [review]
# Please don't submit broken files 🚨
The file is not running on the console. Your blocks are not closed. Please take a look and after that submit a new code review request. Thanks!

File: bubble_sort.rb (hunk 2 above)
Comment (jeton-th) [reply bubble_sort.rb]: I don't think I can because I don't want too iterate the last item.

Comment (heyapricot) [reply bubble_sort.rb]: Aren't `(0...a.length - 1)` and `(0.. a.length)` the same? With the benefit that the second doesn't have an extra operation `(-)` and a magic number?

Comment (jeton-th) [reply bubble_sort.rb]: No they are not, `0..a.length - 1` is same as `0...a.length`
But  `0..a.length` is not same as `0...a.length-1`

**Review by heyapricot — APPROVED** [review]
# Bubblin' 🛁
It is working now! Yay! Good job implementing bubble sort on ruby. No need from my side for further review. Movin' on 🚀

**Pull request merged**